client/src/lib/utils/networkData.ts

```diff
@@ -42,10 +42,7 @@ export const Westend: NetworkData = {
   networkName: "Westend",
   currency: "WND",
   chains: [
-    { name: "Westend", id: -1 },
-    { name: "Collectives", id: 1001 },
-    { name: "BridgeHub", id: 1002 },
-    { name: "People", id: 1004 },
+    { name: "Westend", id: 1000 },
```

Comment:
This is here for compatibility. On the UI the chains dropdown should be totally removed when Westend is selected (or just in general)